cpp/daal/src/threading/service_thread_pinner.cpp

```diff
@@ -37,7 +37,6 @@
         #include "tbb/tbb.h"
         #include <tbb/task_arena.h>
```

Comment:
Potentially it'll be good to remove all of them excluding "tbb/tbb.h"
But let's do it next time 🤪